includes/Admin/Admin_Page.php

```diff
@@ -300,4 +302,28 @@ public function admin_footer() {
 	public function get_hook_suffix() {
 		return $this->hook_suffix;
 	}
+
+	/**
+	 * Gets default check categories to be selected.
+	 *
+	 * @since 1.0.2
+	 *
+	 * @return string Categories separated by double underscores.
+	 */
+	public function get_default_check_categories_to_be_selected() {
```

Comment:
No need for this to be public

Reply:
Yah, I understood. But when I kept private/protected I could not implement unit test for it as it gave fatal error. Any suggestions?

Reply:
We could test it via `render_page()` (like the `test_render_page` tests)